### PR TITLE
set explicit license instead of reference to LICENSE source file

### PR DESCRIPTION
**Pull request by osallou**
* [ X ] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [ ] This PR adds a new recipe.
* [ ] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ X ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/yahmm/meta.yaml [hunk 1]
```diff
@@ -33,7 +33,7 @@ test:
 
 about:
   home: http://pypi.python.org/pypi/yahmm/
-  license: LICENSE.txt
```
Comment (bgruening) [inline recipes/yahmm/meta.yaml]: Should this be license_file?

Comment (osallou) [reply recipes/yahmm/meta.yaml]: could be , but needs to access source. Could be added as a link to source location (http:///), don't know how you usually manage references in source files.

Comment (bgruening) [reply recipes/yahmm/meta.yaml]: LICENSE.txt is indeed in the tarball, so you can change this to license_file:

Comment (osallou) [reply recipes/yahmm/meta.yaml]: added license_info

**Comment by bgruening** [comment]
Thanks @osallou!

**Pull request merged**